nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/support/fixtures/users.js

```diff
@@ -37,5 +37,13 @@ fixtures.users = {
 
 module.exports = function () {
   this.After(() => Promise.all(Object.keys(users)
-      .map((user) => user !== 'Administrator' ? fixtures.users.delete(user) : Promise.resolve())));
+      .map((user) => {
+        if (user !== 'Administrator') {
+          const userWorkspace = `/default-domain/UserWorkspaces/${user}`;
+          nuxeo.repository().delete(userWorkspace).catch(() => {});
+          fixtures.users.delete(user);
```

Comment:
I've also just added a couple of lines to delete user favorites, as running the Permissions test scenario before the Search test scenario creates the same problem: the Favorites of the user are returned as search results.